index.html

```diff
@@ -1,246 +1,283 @@
-</head>
-<body>
-    
+    <link rel="stylesheet" href="style.css" />
+    <link rel="shortcut icon" href="ikony/olx.png" />
+  </head>
+  <body>
     <nav>
-        <a href="link do strony głównej" class="olx">
-            <img src="ikony/olx.png" alt="ikona olx" width="110px" height="70px">
-        </a>
-        <a href="link do wiadomości" class="headline">
-            <img src="ikony/wiadomosci.png" alt="ikona wiadomości" width="18px" height="18px"> <b>Wiadomości</b>
-        </a>
-        <a href="link do obserwowane" class="headline">
-            <img src="ikony/obserwowane.png" alt="ikona obserwowane" width="18px" height="18px">
-        </a>
-        <a href="link do twoje konto" class="headline">
-            <img src="ikony/twojekonto.png" alt="ikona twoje konto" width="18px" height="18px"> <b>Twoje konto</b>
-        </a>
-        <button class="add">Dodaj ogłoszenie</button>
+      <a href="link do strony głównej" class="olx">
+        <img src="ikony/olx.png" alt="ikona olx" width="110px" height="70px" />
+      </a>
+      <a href="link do wiadomości" class="headline">
+        <img
+          src="ikony/wiadomosci.png"
+          alt="wiadomości"
+          width="18px"
+          height="18px"
+        />
+        <b>Wiadomości</b>
+      </a>
+      <a href="link do obserwowane" class="headline">
+        <img
+          src="ikony/obserwowane.png"
+          alt="obserwowane"
+          width="18px"
+          height="18px"
+        />
+      </a>
+      <a href="link do twoje konto" class="headline">
+        <img
+          src="ikony/twojekonto.png"
+          alt="twoje konto"
+          width="18px"
+          height="18px"
+        />
+        <b>Twoje konto</b>
+      </a>
+      <button class="add">Dodaj ogłoszenie</button>
     </nav>
 
     <h1>
-            <input type="search">    
-        <select name="" id="">
-            <img src="ikony/place.png" alt="" class="place" width="25px"> //?
-            <option value="">Cała Polska</option>
-            <option value="">Dolnośląskie</option>
-            <option value="">Kujawsko-pomorskie</option>
-            <option value="">Lubelskie</option>
-            <option value="">Lubuskie</option>
-            <option value="">Łudzkie</option>
-            <option value="">Małopolskie</option>
-            <option value="">Mazowieckie</option>
-            <option value="">Opolskie</option>
-            <option value="">Podkarpackie</option>
-            <option value="">Podlaskie</option>
-            <option value="">Pomorskie</option>
-            <option value="">Śląskie</option>
-            <option value="">Świętokrzyskie</option>
-            <option value="">Warmińsko-mazurskie</option>
-            <option value="">Wielkopolskie</option>
-            <option value="">Zachodniopomorskie</option>
-        </select>
+      <input type="search" />
+      <select name="" id="">
+        <img src="ikony/place.png" alt="" class="place" width="25px" />
+        //?
```

Comment:
Co to jest?

Reply:
miałam wcześniej z tym problem i zapomniałam usunąć :P już usuwam